src/Sylius/Bundle/TaxonomyBundle/Form/Type/TaxonType.php

```diff
@@ -32,6 +33,10 @@ public function buildForm(FormBuilderInterface $builder, array $options): void
                 'entry_type' => TaxonTranslationType::class,
                 'label' => 'sylius.form.taxon.name',
             ])
+            ->add('enabled', CheckboxType::class, [
```

Comment:
Shouldn't it be `true` as default?

Reply:
I followed what has been done for the enabled field in other entities and it is as this: no default value (as it true/checked as default) and required=>false. I am ok in changing this btw